src/main/java/org/spongepowered/mod/mixin/entity/player/MixinEntityPlayerMP.java

```diff
@@ -55,7 +55,7 @@
 @NonnullByDefault
 @Mixin(EntityPlayerMP.class)
 @Implements(@Interface(iface = Player.class, prefix = "playermp$"))
-public abstract class MixinEntityPlayerMP extends EntityPlayer implements CommandSource {
+public abstract class MixinEntityPlayerMP extends SpongeUser implements CommandSource {
 
```

Comment:
It will cause issues if you **don't** switch it back, the mixin will not work if you arbitrarily change its superclass.